### PR TITLE
[docs]: translate hertz/tutorials/tutorials/framework-exten/log

### PR DESCRIPTION
**Pull request by Hao-Wu**
translate hertz/tutorials/tutorials/framework-exten/log
[https://github.com/cloudwego/hertz/issues/81](https://github.com/cloudwego/hertz/issues/81)

### CODE REVIEW

File: content/en/docs/hertz/tutorials/framework-exten/log.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+weight: 2
+description: >
+
+---
+
+Hertz provides logger extension, and the interface is defined in `pkg/common/hlog`. 
+
+## Interface Defination
+
+In Hertz, the interfaces `Logger`、`CtxLogger`、`FormatLogger` are defined in `pkg/common/hlog`, and these interfaces are used to output logs in different ways, and a Control interface is defined to control the logger.
```
Comment (GuangmingLuo) [inline content/en/docs/hertz/tutorials/framework-exten/log.md:14]: `Logger`, `CtxLogger` and `FormatLogger`

-> use , instead of 、

Comment (Hao-Wu) [reply content/en/docs/hertz/tutorials/framework-exten/log.md]: resolved

File: content/en/docs/hertz/tutorials/framework-exten/log.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+// FullLogger is the combination of Logger, FormatLogger, CtxLogger and Control.
+type FullLogger interface {
+   Logger
+   FormatLogger
+   CtxLogger
+   Control
+}
+```
+
+Note that the default logger makes use of the standard library `log.Logger` as its underlying output. So the filenames and line numbers shown in the log messages depend on the settings of call depth. Thus wrapping the implementation of hlog may causes inaccuracies for these two value.
```
Comment (GuangmingLuo) [inline content/en/docs/hertz/tutorials/framework-exten/log.md:27]: causes -> cause
two value -> two values

Comment (Hao-Wu) [reply content/en/docs/hertz/tutorials/framework-exten/log.md]: Resolved, and updated the same places in Kitext doc

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**